doc/connectivity/bluetooth/api/audio/bluetooth-le-audio-arch.rst

```diff
@@ -59,6 +59,292 @@ GAF has been implemented in Zephyr with the following structure.
 
    Zephyr Generic Audio Framework
 
+
+Dependency on GATT
+==================
+
+The following tables show the required sub-procedures from GATT for each of the
+LE Audio profiles and services.
+
+.. table:: GATT sub-procedures for LE Audio profiles
+   :widths: auto
+
+   +------------------------------------------+-----+-----+-----+------+------+-----+-----+------+-----+------+
+   | GATT sub-procedure                       | BAP | CCP | CAP | CSIP | GMAP | HAP | MCP | MICP | PBP | TMAP |
+   +==========================================+=====+=====+=====+======+======+=====+=====+======+=====+======+
+   | Exchange MTU                             | M   |     | I   |      |      | I   |     |      |     |      |
+   +------------------------------------------+-----+-----+-----+------+------+-----+-----+------+-----+------+
+   | Discover all primary services            | C.1 | C.1 | I   | C.1  | C.1  | C.1 | C.1 | C.1  |     | C.1  |
```

Comment:
When you use inherited (`I`) the same way for both mandatory, optional and conditional, then it's not possible to know if it's mandatory, optional or conditional for e.g. CAP.